packages/create-toolpad-app/index.ts

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/env node
+
+import chalk from 'chalk';
+import { execaCommand } from 'execa';
+import inquirer from 'inquirer';
+import fs from 'fs';
+import { getPackageManager } from './utils/getPackageManager';
+
+// Detect the package manager
+const packageManager = getPackageManager();
+
+// Create a new directory and initialize a new project
+const scaffoldProject = async () => {
+  const name = await inquirer.prompt([
+    {
+      type: 'input',
+      name: 'projectName',
+      message: 'Enter the name of your project:',
+      default: 'my-app',
+    },
+  ]);
+
+  // eslint-disable-next-line no-console
+  console.log(`Creating a new MUI Toolpad project in ${chalk.blue(name.projectName)}`);
+  try {
+    fs.mkdirSync(name.projectName);
+    process.chdir(name.projectName);
+
+    const packageJson = {
+      name: name.projectName,
+      version: '0.1.0',
+      private: true,
+      scripts: {
+        dev: 'toolpad dev',
+        build: 'toolpad build',
+        start: 'toolpad start',
+      },
+    };
+
+    fs.writeFileSync('package.json', JSON.stringify(packageJson, null, 2));
+    return name.projectName;
+  } catch (error) {
+    if (fs.existsSync(name.projectName)) {
+      console.error(
+        `Directory ${chalk.red(name.projectName)} already exists. Please provide a different name.`,
+      );
+    }
+    return null;
+  }
+};
+
+// Install the dependencies
+const installDeps = async () => {
+  const installVerb = packageManager === 'yarn' ? 'add' : 'install';
+  const command = `${packageManager} ${installVerb} @mui/toolpad @mui/toolpad-core`;
```

Comment:
Won't this install the latest versions? We might want to install specific versions to keep things a bit more under control.
Also if we end up wanting to install more dependencies the pre-generated package.json approach I mentioned might work best.